openfisca_nsw_safeguard/tests/ESS_PDRS_Estimator/WH1_F16_electric_PDRSAug24/WH1_F16_electric_PDRSAug24_installation_final_activity_eligibility.yaml

```diff
@@ -199,5 +210,129 @@
         False,
         False,
         False,
+        False,
         False
+      ]
+
+- name: Test WH1 final activity eligibility 2
+  period: 2024
+  absolute_error_margin: 0
+  input:  
+    WH1_F16_electric_PDRSAug24__equipment_replaced:
+      [ 
+        true,
+        true,
+        true,
+        true,
+        true,
+        true
+      ]
+    WH1_F16_electric_PDRSAug24__existing_equipment_removed:
+      [ 
+        true,
+        true,
+        true,
+        true,
+        true,
+        true
+      ]
+    WH1_F16_electric_PDRSAug24__equipment_installed_on_site:
+      [ 
+        true,
+        true,
+        true,
+        true,
+        true,
+        true
+      ]
+    WH1_F16_electric_PDRSAug24__qualified_install_removal:
+      [ 
+        true,
+        true,
+        true,
+        true,
+        true,
+        true
+      ]
+    WH1_F16_electric_PDRSAug24__engaged_ACP:
+      [ 
+        true,
+        true,
+        true,
+        true,
+        true,
+        true
+      ]
+    WH1_F16_electric_PDRSAug24__minimum_payment:
+      [ 
+        true,
+        true,
+        true,
+        true,
+        true,
+        true
+      ]
+    WH1_F16_electric_PDRSAug24__building_BCA_not_class_1_or_4:
+      [ 
+        true,
+        false,
+        true,
+        false,
+        true,
+        true
+      ]
+    WH1_F16_electric_PDRSAug24__4234_certified:
+      [ 
+        true,
+        true,
+        true,
+        true,
+        true,
+        true
+      ]
+    WH1_F16_electric_PDRSAug24__scheme_admin_approved:
+      [ 
+        true,
+        true,
+        true,
+        true,
+        true,
+        true
+      ]
+    WH1_F16_electric_PDRSAug24__minimum_annual_energy:
+      [ 
+        true,
+        true,
+        true,
+        true,
+        true,
+        true
+      ]
+    WH1_F16_electric_PDRSAug24__storage_volume:
+      [ 
+        less_than_425_L,
+        less_than_425_L,
+        425 - 700 litres,
+        425 - 700 litres,
+        More than 700 litres,
+        More than 700 litres
+      ]
+    WH1_F16_electric_PDRSAug24__certified:
+      [ 
+        true,
+        false,
+        true,
+        false,
+        true,
+        true
+      ]
+  output:
+    WH1_F16_electric_PDRSAug24__installation_replacement_final_activity_eligibility:
+      [
+        true,
+        False,
```

Comment:
F is uppercase, can you change?

Reply:
i have updated this on a new branch, pushed on (PR 441)